sdk/workloads/azure-mgmt-workloadssapvirtualinstance/azure/mgmt/workloadssapvirtualinstance/aio/operations/_operations.py

```diff
@@ -16,60 +19,861 @@
+         Required.
+        :type resource_group_name: str
+        :param sap_virtual_instance_name: The name of the Virtual Instances for SAP solutions resource.
+         Required.
+        :type sap_virtual_instance_name: str
+        :param properties: Request body to update a Virtual Instance for SAP solutions resource. Is one
+         of the following types: UpdateSAPVirtualInstanceRequest, JSON, IO[bytes] Required.
+        :type properties:
+         ~azure.mgmt.workloadssapvirtualinstance.models.UpdateSAPVirtualInstanceRequest or JSON or
+         IO[bytes]
+        :return: An instance of AsyncLROPoller that returns SAPVirtualInstance. The SAPVirtualInstance
+         is compatible with MutableMapping
+        :rtype:
+         ~azure.core.polling.AsyncLROPoller[~azure.mgmt.workloadssapvirtualinstance.models.SAPVirtualInstance]
+        :raises ~azure.core.exceptions.HttpResponseError:
+        """
+        _headers = case_insensitive_dict(kwargs.pop("headers", {}) or {})
+        _params = kwargs.pop("params", {}) or {}
+
+        content_type: Optional[str] = kwargs.pop("content_type", _headers.pop("Content-Type", None))
+        cls: ClsType[_models.SAPVirtualInstance] = kwargs.pop("cls", None)
+        polling: Union[bool, AsyncPollingMethod] = kwargs.pop("polling", True)
+        lro_delay = kwargs.pop("polling_interval", self._config.polling_interval)
+        cont_token: Optional[str] = kwargs.pop("continuation_token", None)
+        if cont_token is None:
+            raw_result = await self._update_initial(
+                resource_group_name=resource_group_name,
+                sap_virtual_instance_name=sap_virtual_instance_name,
+                properties=properties,
+                content_type=content_type,
+                cls=lambda x, y, z: x,
+                headers=_headers,
+                params=_params,
+                **kwargs
+            )
+            await raw_result.http_response.read()  # type: ignore
+        kwargs.pop("error_map", None)
+
+        def get_long_running_output(pipeline_response):
+            response = pipeline_response.http_response
+            deserialized = _deserialize(_models.SAPVirtualInstance, response.json())
+            if cls:
+                return cls(pipeline_response, deserialized, {})  # type: ignore
+            return deserialized
+
+        path_format_arguments = {
+            "endpoint": self._serialize.url("self._config.base_url", self._config.base_url, "str", skip_quote=True),
+        }
+
+        if polling is True:
+            polling_method: AsyncPollingMethod = cast(
+                AsyncPollingMethod, AsyncARMPolling(lro_delay, path_format_arguments=path_format_arguments, **kwargs)
+            )
+        elif polling is False:
+            polling_method = cast(AsyncPollingMethod, AsyncNoPolling())
+        else:
+            polling_method = polling
+        if cont_token:
```

Comment:
Compared with last release https://github.com/Azure/azure-sdk-for-python/blob/b53b7bbb255d0588bef2ec61cc23d7db8d2fbcb7/sdk/workloads/azure-mgmt-workloadssapvirtualinstance/azure/mgmt/workloadssapvirtualinstance/aio/operations/_sap_virtual_instances_operations.py#L594-L599, this LRO operation changes lro options